summIApp/utils_validations.py

```diff
@@ -1,23 +1,38 @@
 from PIL import Image
 import logging
 from pathvalidate import sanitize_filename
+# regex library added
+import re
+logger = logging.getLogger("django")
 
 
-logger = logging.getLogger("django")
 def validate_file(file):
     try:
+        # Opening and closing files manually can cause errors if you forget to close the file.
+        # Using context managers (with statements) ensures that the file is automatically closed after the block of code finishes executing.
         img = Image.open(file)
-
-        return img.format.upper() in ["PNG", "JPG", "JPEG", "WEBP", "TIFF"]
+        with Image.open(file) as img:
+            return img.format.upper() in ["PNG", "JPG", "JPEG", "WEBP", "TIFF"]
     except Exception as e:
         logger.error(str(e))
         return False
 
-def strip_html(name):
-    sanitized_filename = ""
+
+def strip_html(name, max_length=100):
     try:
-        sanitized_filename = sanitize_filename(name)
```

Comment:
Please be sure to use sanitize_filename function. As this func removes HTML and unnecessary characters in the filename